Assets/PatchKit Patcher/Editor/CustomBuilding.cs

```diff
@@ -78,24 +117,40 @@ private static string PatcherExecutableName(BuildTarget target)
             }
         }
 
-        private static void Build(BuildTarget target)
+        private static void BuildFromBuildSettings(BuildTarget target)
         {
-            if (!EditorSceneManager.SaveCurrentModifiedScenesIfUserWantsTo())
-            {
-                return;
-            }
-
             string[] scenePaths = EditorBuildSettings.scenes
                 .Where(s => s.enabled)
                 .Select(s => s.path)
                 .ToArray();
 
-            if (scenePaths.Count() == 0)
+            BuildScenes(target, scenePaths);
+        }
+
+        private static void BuildFromCurrentScene(BuildTarget target)
+        {
+            string[] scenePaths = {SceneManager.GetActiveScene().path};
+
+            BuildScenes(target, scenePaths);
+        }
+
+        private static void BuildScenes(BuildTarget target, string[] scenePaths)
+        {
+            if (!EditorSceneManager.SaveCurrentModifiedScenesIfUserWantsTo())
+            {
+                return;
+            }
+
+            if (!scenePaths.Any())
             {
                 EditorUtility.DisplayDialog("Error", "Add or enable scenes to be included in the Build Settings menu.", "Ok");
                 return;
             }
 
+            PlayerSettings.defaultScreenWidth = 600;
+            PlayerSettings.defaultScreenHeight = 400;
+            PlayerSettings.defaultIsFullScreen = false;
```

Comment:
One thing is missing here.
Patcher won't build if PlayerSettings API Compatibility Level is set to something else than .NET 2.0 Subset. I'm not 100% sure about that, so please double check it.